.devcontainer/Dockerfile

```diff
@@ -2,9 +2,8 @@
 
 FROM ubuntu:mantic
 
-COPY install-dependencies.sh /tmp/
-
-RUN apt-get update && apt-get install -y \
+RUN --mount=type=bind,source=./install-dependencies.sh,target=./install-dependencies.sh \
```

Comment:
The patch comment lacks motivation of this change

Reply:
I thought it reduces layers - since we do not copy the file over to the container.

Reply:
> I thought it reduces layers - since we do not copy the file over to the container.

yep, it's right